Dockerfile

```diff
@@ -1,6 +1,28 @@
+# Use a UUID as placeholder value to have a unique string to replace. 
+ARG BASE_URL_PLACEHOLDER=189b303e-37a0-4f6f-8c0a-50333bc3c36e
```

Comment:
I'm honestly not an expert when it comes to Docker, but why is `BASE_URL_PLACEHOLDER` defined three times? Isn't it better to define these arguments once at the top like:

```
ARG BASE_URL_PLACEHOLDER=189b303e-37a0-4f6f-8c0a-50333bc3c36e
ARG ENTRYPOINT_SCRIPT=/docker-entrypoint.d/set-public-url.sh
```

and remove other occurrences with `ARG`?

Reply:
The variable value can be defined centrally, but it's unfortunately necessary to repeat the variable name in each stage :/